generate/testdata/queries/foo.go

```diff
@@ -0,0 +1,69 @@
+// Code generated by github.com/Khan/genqlient, DO NOT EDIT.
```

Comment:
Is this used? Maybe left over from your testing?

Reply:
Oups. going to remove it